microsoft-edge/webview2/roadmap.md

```diff
@@ -69,6 +69,12 @@ You can access WebView2 controls in your applications using [Windows UI Library
 XAML Island support requires additional work and may be considered for future releases.
 
 
+<!-- ------------------------------ -->
+#### Xbox
+
+WebView2 is now available on Xbox devices. 
```

Comment:
```suggestion
WebView2 is available on Xbox devices. 
```

References to time-based things, like the word "now", are not usually needed and often force us to update docs after a while. I suggest just removing it.

Reply:
keeping "now"